src/components/Placeholder/Placeholder.js

```diff
@@ -0,0 +1,25 @@
+// Import Styling
+import './Placeholder.scss';
+
+// Placeholder string
+const Placeholder = () => (`
+  <div class="placeholder">
+    <p class="placeholder--medium">&nbsp;</p>
```

Comment:
I think these would actually be elements not modifiers so `placeholder__medium` would make more sense to stay consistent with BEM.

Reply:
Yeah totally.